src/analytics/interactions/interaction_mapper.py

```diff
@@ -70,3 +74,26 @@ def map_from_amplitude_event(amplitude_event: AmplitudeEvent) -> UserInteraction
         personalize_rec_id=amplitude_event.personalize_rec_id,
         impression=amplitude_event.impression,
     )
+
+
+def map_from_comment(comment) -> UserInteractions:
+    """
+    Map a RhCommentModel record to a UserInteractions instance (not saved to database).
+    """
+    event_type = (
+        PEER_REVIEW_CREATED if comment.comment_type == PEER_REVIEW else COMMENT_CREATED
+    )
+
+    content_type = ContentType.objects.get_for_model(RhCommentModel)
```

Comment:
Nit: Since the content type is static, we usually try to limit the number of times for these lookups by keeping an instance variable and a lazy load method, e.g., https://github.com/ResearchHub/researchhub-backend/blob/6977b829aba2c8c6d726b70b07c9a2c90e0e4d65/src/paper/ingestion/mappers/preprints/chemrxiv.py#L38.

Reply:
very good suggestion!